src/main/scala/tilelink/Broadcast.scala

```diff
@@ -38,33 +132,38 @@ class TLBroadcast(params: TLBroadcastParams)(implicit p: Parameters) extends Laz
     clientFn  = { cp =>
       cp.v1copy(clients = Seq(TLMasterParameters.v1(
         name     = "TLBroadcast",
+        /* Add 2 bits encoded in [[TLBroadcastConstants]] to record transaction type for each sourceId in clients. */
         sourceId = IdRange(0, 1 << log2Ceil(cp.endSourceId*4)))))
     },
     managerFn = { mp =>
       mp.v1copy(
         endSinkId  = params.numTrackers,
         managers   = mp.managers.map { m =>
-          // We are the last level manager
+          /** This is the last level coherence manager, so it cannot be probed by mangers. */
           require (!m.supportsAcquireB)
-          // We only manage addresses which are uncached
+          /* We only manage addresses which are uncached. */
           if (m.regionType == RegionType.UNCACHED) {
-            // The device had better support line transfers
+            /* The device had better support line transfers.
+             * @todo why?
+             */
             val lowerBound = max(m.supportsPutFull.min, m.supportsGet.min)
             require (!m.supportsPutFull || m.supportsPutFull.contains(params.lineBytes), s"${m.name} only supports PutFull(${m.supportsPutFull}), which does not include ${params.lineBytes}")
             require (!m.supportsGet     || m.supportsGet    .contains(params.lineBytes), s"${m.name} only supports Get(${m.supportsGet}), which does not include ${params.lineBytes}")
             m.v1copy(
               regionType         = RegionType.TRACKED,
               supportsAcquireB   = TransferSizes(lowerBound, params.lineBytes),
+              /* dirty data write back to ddr in a transaction. */
               supportsAcquireT   = if (m.supportsPutFull) TransferSizes(lowerBound, params.lineBytes) else TransferSizes.none,
               alwaysGrantsT      = false,
-              // truncate supported accesses to lineBytes (we only ever probe for one line)
+              /* truncate supported accesses to lineBytes (we only ever probe for one line). */
               supportsPutFull    = TransferSizes(m.supportsPutFull   .min, min(m.supportsPutFull   .max, params.lineBytes)),
               supportsPutPartial = TransferSizes(m.supportsPutPartial.min, min(m.supportsPutPartial.max, params.lineBytes)),
               supportsGet        = TransferSizes(m.supportsGet       .min, min(m.supportsGet       .max, params.lineBytes)),
               supportsHint       = TransferSizes(m.supportsHint      .min, min(m.supportsHint      .max, params.lineBytes)),
               supportsArithmetic = TransferSizes(m.supportsArithmetic.min, min(m.supportsArithmetic.max, params.lineBytes)),
               supportsLogical    = TransferSizes(m.supportsLogical   .min, min(m.supportsLogical   .max, params.lineBytes)),
-              fifoId             = None // trackers do not respond in FIFO order!
+              /* trackers do not respond in FIFO order! */
+              fifoId             = None
```

Comment:
I am not sure it is necessarily better to change these single line comments to be separate line with `/* */`? They won't show up as scala-doc anyways, right?

Reply:
i would specifically ask that we NOT make this change